docs/replatforming.md

```diff
@@ -61,6 +61,11 @@ In the above front matter:
 - `order: 3` explicitly sets the order the page should appear in the sidenav (in this case it'll be first).
 - `title: Active voice` controls what text is displayed in the sidenav. This field is optional, and if it’s omitted the `key` value will be displayed.
 
+### Sticky sidenavs
+Use `sticky_sidenav: true` to stick the sidenav to the top of the window when scrolling.
+
+### Subnavs
+You can use the existing `subnav:` options in the original file's front matter to generate a subnav with the current page's anchor links. To prevent plugin overloading, ensure the `parent` and `key` values are different.
```

Comment:
Suggestion: I am not sure if I totally understand what you mean by "plugin overloading" here. Maybe "To prevent build issues" or "To prevent errors in `eleventyNavigation`" ?

Reply:
Sounds good!